includes/admin/class-amp-post-meta-box.php

```diff
@@ -342,7 +342,7 @@ public static function get_featured_image_dimensions() {
 	 */
 	public function render_status( $post ) {
 		$verify = (
-			isset( $post->ID )
+			! empty( $post->ID )
```

Comment:
Error in PHPstan

```
------ --------------------------------------------------------- 
  Line   includes/admin/class-amp-post-meta-box.php               
 ------ --------------------------------------------------------- 
  345    Property WP_Post::$ID (int) in isset() is not nullable.  
 ------ --------------------------------------------------------- 
```

Because it assumes `$post` will always be `WP_Post` type as per function doc.

Reply:
And the `$ID` member of `WP_Post` is defined as `int`:

```php
final class WP_Post {

	/**
	 * Post ID.
	 *
	 * @since 3.5.0
	 * @var int
	 */
	public $ID;
...
```

Even though it doesn't have an initial value and thus will actually be `null` if not set. 

So yeah, another case where phpdoc in WordPress seems incomplete.